src/markitdown/_markitdown.py

```diff
@@ -146,6 +146,17 @@ def __init__(self, title: Union[str, None] = None, text_content: str = ""):
         self.title: Union[str, None] = title
         self.text_content: str = text_content
 
+    def save(self, file_path: str, encoding: str = "utf-8") -> None:
+        """
+        Save the converted document result `text_content` to a file.
+
+        params:
+            file_path: The path to save the document result to.
+            encoding: The encoding to use when writing the document.
+        """
+        with open(file_path, "w", encoding=encoding) as f:
+            f.write(self.text_content)
+
```

Comment:
it serves the same purpose as #116. could you refactor the code to use a single function for better reusability? 
- #116

Reply:
sure @l-lumin,

I think it makes more sense to have the function to save the result into a file inside the `DocumentConverterResult`.

* I refactored the CLI to use this function.
* I also added an `encoding` parameter to the arguments.

Reply:
Thank you. I think so too. Look good